src/test/java/org/cactoos/io/JoinedTest.java

```diff
@@ -52,4 +53,24 @@ public void joinsOk() throws IOException {
             new TextHasString("firstsecondthird")
         );
     }
+
+    /**
+     * Must join inputs of the iterable in the given order.
+     */
+    @Test
+    public void fromIterable() {
+        MatcherAssert.assertThat(
+            "Cannot join iterable of inputs",
```

Comment:
@ilyakharlamov Change "Cannot" to "Can't". Cannot is not wrong, but "Can't"  is often used here.